op5build/ci_config.yml

```diff
@@ -0,0 +1,11 @@
+pre:
+  steps: |
+    set -ex
+    cd src
+    python3 -m venv .venv  # poetry will find and use .venv
+    source .venv/bin/activate
+    pip install --upgrade pip==21.1.3
+    pip install poetry
+    poetry env info
```

Comment:
Was a little curious why we would run this, but I assume it's for having additional info in build logs?

Reply:
It's not necessary, no. But since it doesn't hurt, I like to have handy info in build log in case something happens, or in case one is unsure of what virtual env is used (since it's a little bit magic here, that poetry will use the `.venv/` if it exists...